web/src/main/java/org/mskcc/cbio/oncokb/api/websocket/CurationValidationApiController.java

```diff
@@ -341,8 +363,59 @@ public JSONArray getTruncatingMutationsNotUnderTSG() throws ApiException {
         return data;
     }
 
+    public JSONArray getActionableVariantsNotOncogenic() throws ApiException {
+        JSONArray data = new JSONArray();
+        List<Alteration> alterations = AlterationUtils.getAllAlterations();
+        List<String> allowedOncogenicities = new ArrayList<>(Arrays.asList(Oncogenicity.YES.getOncogenic(), Oncogenicity.LIKELY.getOncogenic(), Oncogenicity.RESISTANCE.getOncogenic()));
+        for (Alteration alteration : alterations) {
+            Query query = new Query(
+                null,
+                DEFAULT_REFERENCE_GENOME,
+                alteration.getGene().getEntrezGeneId(),
+                alteration.getGene().getHugoSymbol(),
+                alteration.getAlteration(),
+                null,
+                null,
+                null,
+                null,
+                alteration.getProteinStart(),
+                alteration.getProteinEnd(),
+                null);
+            IndicatorQueryResp response = IndicatorUtils.processQuery(query, null, false,null, false);
+            if (!allowedOncogenicities.contains(response.getOncogenic())) {
+                if (response.getHighestSensitiveLevel() != null || response.getHighestResistanceLevel() != null) {
+                    String hugoSymbol = alteration.getGene().getHugoSymbol();
+                    List<IndicatorQueryTreatment> treatments = response.getTreatments();
+                    for (IndicatorQueryTreatment treatment: treatments) {
+                        for (String altString: treatment.getAlterations()) {
+                            TumorType tumorTypeModel = new TumorType(treatment.getLevelAssociatedCancerType());
+                            Set<TumorType> excludedTumorTypeModels =  treatment.getLevelExcludedCancerTypes().stream().map(excludedTT -> {
+                                return new TumorType(excludedTT);
+                            }).collect(Collectors.toSet());
+                            String tumorName = TumorTypeUtils.getTumorTypesNameWithExclusion(Collections.singleton(tumorTypeModel), excludedTumorTypeModels);
+                            
+                            StringBuilder errorMessage = new StringBuilder();
+                            errorMessage.append("Is ");
+                            errorMessage.append(response.getOncogenic());
+                            errorMessage.append(", but has ");
+                            errorMessage.append(treatment.getLevel().toString());
+                            errorMessage.append(" treatment: ");
+                            List<String> drugList = treatment.getDrugs().stream().map(drug -> {
+                                return drug.getDrugName();
+                            }).collect(Collectors.toList());
+                            errorMessage.append(StringUtils.join(drugList, " + "));
```

Comment:
Essentially when the variant has multiple treatments, the `is inconclusive` message will be repeated?

Reply:
Yeah it could be possible `Gene / Alteration / CT` is repeated if there is are multiple treatments.